pkg/controller/core/workload_controller.go

```diff
@@ -614,6 +614,8 @@ func (r *WorkloadReconciler) Create(e event.TypedCreateEvent[*kueue.Workload]) b
 			log.V(2).Info("ignored an error for now", "error", err)
 		}
 		return true
+	} else {
+		r.queues.QueueSecondPassIfNeeded(ctx, e.Object)
 	}
```

Comment:
```suggestion
	}		
	r.queues.QueueSecondPassIfNeeded(ctx, e.Object)
```
Nit

Reply:
Yes, this would work, but would mean unnecessary work for double checking the quota reserved condition. I think in case of node replacements we will also only schedule second pass if quota is reserved. So, it seems ok for the foreseeable future.

Reply:
Actually, let me move the check to the bottom, to align with handling of Update()

Reply:
Done